docs/concepts/insert.rst

```diff
@@ -167,7 +167,7 @@ construct the ``DEFAULT`` expression::
 .. warning::
    Also note PostgreSQL's syntax rules mean that ``DEFAULT``` can only appear
    in ``INSERT``` expressions whose rows are specified using ``VALUES``. This
-   means that the ``rows`` field of your ``Insert`` record doesn't look like
+   means if that the ``rows`` field of your ``Insert`` record doesn't look like
```

Comment:
This doesn't look correct to me. Did you mean:

```suggestion
   means that if the ``rows`` field of your ``Insert`` record doesn't look like
```

Reply:
Bah, you're right.